src/hooks/useNavState.js

```diff
@@ -0,0 +1,11 @@
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+
+export function useNavState() {
+  const currentUrl = useLocation()
+    .pathname.replace(/\d/, "")
+    .replace(/^\/+|\/+$/g, "");
+  const [selectedmainNav, setselectedMainNav] = useState("Watching");
```

Comment:
```suggestion
  const [selectedMainNav, setSelectedMainNav] = useState("Watching");
```
camelCase 표기법을 제대로 지켜주세요~